### ISSUE TITLE
Adding eraser method to API

### ISSUE THREAD

**Issue by antonioquepasa** [yiom/sketchpad#21]
Hello guys,

Thanks for creating and sharing such a great library. Easy as pie and incredibly useful.

Any chance or plan to add an eraser method to the API?

Cheers and keep up with the great work,

Antoine

**Comment by nihey** [comment]
Hello Antoine,

Sure, I was off for some time to fix up some personal issues, but I'll be working on some issues here after this weekend. Feedback is greatly appreciated.

Nihey

**Comment by jeanleonino** [comment]
Antoine, thanks for the feedback. What do you mean by erase? You do have
undo/redo and you can pick the color you're painting, thus "erasing" (in
some editors eraser is just a white brush).

**Comment by antonioquepasa** [comment]
Can I select a transparent color that would erase any color that would be on the canvas, thus act as an eraser? If so, that would totally do the trick.

**Comment by jeanleonino** [comment]
@antonioquepasa yeah, there are work arounds but it really isn't implemented right now, probably for version 1.0 :-) just labeled as `feature-request` to mark it.
